src/encoded/region_search.py

```diff
@@ -265,7 +264,7 @@ def region_search(context, request):
         'notification': '',
         'filters': []
     }
-    principals = effective_principals(request)
+    principals = request.effective_principals
```

Comment:
this , I assume is a pyramid upgrade issue?

Reply:
yea, That is from lrowe.  Did you look at the commit history?  That explains some of the changes and there is a specific one for lrowes updates.